script/generate-constant-typings.js

```diff
@@ -0,0 +1,41 @@
+// This script will read constants from `.js` files and generate `.d.ts` files.
+
+var fs = require('fs')
+var path = require('path')
+
+function generateTyping(name, paths) {
+  var constants = JSON.parse(JSON.stringify(require('../' + paths.join('/'))))
```

Comment:
```suggestion
  var constants = require('../' + paths.join('/'))
```

Any reason for it to be deep cloned?